espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/AndroidLogger.java

```diff
@@ -47,5 +47,9 @@ public void debug(Object... messages) {
         android.util.Log.d(TAG, toString(messages));
     }
 
+    public void warn(Object... messages) {
+        android.util.Log.w(TAG, toString(messages));
```

Comment:
will this ultimately show up in the driver logs? that's probably where we will want it...

Reply:
@jlipps yeah, these are the logs that show up in the Appium logs.